src/Oscoin/Storage/Block/Memory.hs

```diff
@@ -0,0 +1,58 @@
+module Oscoin.Storage.Block.Memory
+    ( newBlockStoreIO
+    ) where
+
+import           Oscoin.Prelude
+
+import           Oscoin.Crypto.Blockchain hiding (lookupTx)
+import           Oscoin.Crypto.Blockchain.Block (Block)
+import qualified Oscoin.Crypto.Hash as Crypto
+import qualified Oscoin.Storage.Block.Abstract as Abstract
+import qualified Oscoin.Time.Chrono as Chrono
+
+import qualified Data.List as List
+
+newBlockStoreIO
```

Comment:
The `IO` suffix kind of suggests that there is a `newBlockStore` function :confused:

Reply:
Yes, this is a tad unfortunate. This comes out of consistency with `ContentStore` and `HashStore`. I’m open to suggestions.

Reply:
I guess I'd leave it as-is for now :+1: and we can revisit in the future if we want.